### PR TITLE
docstring edits

### PR DESCRIPTION
**Pull request by Bonface-Osoro**
1. Completed doctstrings.
2. Wrote in the docstring the equation with units to provide greater explanation.
3. Removed existing `path loss` and `log normal dist` functions from sim.py
4. Removed `path lost equation` from runner and added it to the capacity.py.
5. Splitted sim.py into multiple emissions, capacity and cost scripts for clarity.
6. Corrected code content, Docstrings, formatted and removed redundant code, functions and metrics.

### CODE REVIEW

File: src/saleos/capacity.py [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+    """ 
+    This function calculates the monthly 
+    traffic assuming the lifespan of all 
+    constellations is 5 years and 20% 
+    accounting for traffic taking place 
+    in the busiest hour of the day.
+
+    Conversion of Mbps to monthly traffic in GB. 
+
+    Monthly traffic (GB) = (
```
Comment (edwardoughton) [inline src/saleos/capacity.py:477]: This is not filled out yet, please can you add in a much better description? I was struggling to understand it. Thanks

File: src/saleos/capacity.py [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+        (Capacity_Mbps / 12 * 5) /  # 12 months in 1 year over 5 years
+        (8000 * #
+        (1 / 30 ) * # 30 days of the month
+        (1 / 3600) * # seconds in a 1 hour
+        (20 / 100) #
```
Comment (edwardoughton) [inline src/saleos/capacity.py:482]: Not sure what this represents, or what the units are here.

**Review by edwardoughton — CHANGES_REQUESTED** [review]
Good work - I only have two comments on the function in the capacity module. Please could you clarify the units/equation there? Adding in a citation for the path loss is also useful e.g., this function does x and y and is based on Smith (2020). This just helps to support the reviewer, rather than leaving them with queries/questions.

File: src/saleos/capacity.py (hunk 1 above)
Comment (Bonface-Osoro) [reply src/saleos/capacity.py]: Resolved

File: src/saleos/capacity.py (hunk 2 above)
Comment (Bonface-Osoro) [reply src/saleos/capacity.py]: Resolved

**Comment by Bonface-Osoro** [comment]
Referenced capacity and emission equations/model for clarity to the reviewers

**Pull request merged**